### ISSUE TITLE
-webkit-overflow-scrolling:touch with dropdowns produces flashes and hidings in other elements

### ISSUE THREAD

**Issue by sarbona** [Semantic-Org/Semantic-UI#3868]
I have been struggling for some days with dropdowns in ios (ipad) because whenever I clicked a dropdown many other elements (labels, buttons, dropdowns and even other elements) started to flash and many times they remain hidden after some flashing.

Because my code was big, I have been trying to simplify the code too few lines during days, and at least I have found what is causing the malfunction.

Basically, the problem only appears in ios using dropdowns inside a div with “-webkit-overflow-scrolling:touch” .

You can try the code in: http://www.haiku-media.com/Resources/Tests/Bug/Bug.html

**Comment by jlukic** [comment]
Thanks for the report. This is well defined. I'll try to debug on my iphone.

**Comment by sarbona** [comment]
I have been able to reproduce the problem in an ipad with ios 9.3 and in an iPhone 5 with ios 8.3.

Because the example has only one label and one dropdown, the label does not hide at all, but it flashes.

In my developement forms where I have dozens of fields (labels, dropdowns, inputs, buttons...) after some flashings many elements remain hidden after the flash and the only way to show them again, is to force a full screen repaint with another component, but not always working.

Please, ask for any other information that you may need to help you reproduce the problem.

Thank you.
